webui/src/Views/PeriodicRunsView.elm

```diff
@@ -104,10 +104,19 @@ row instanceId instanceTasks periodicRun =
                                 ]
                                 [ JobStatusView.view "" periodicRun.status
                                 , text " "
-                                , iconButton
-                                    "btn btn-default btn-xs"
-                                    "glyphicon glyphicon-stop"
-                                    "Stop Instance"
+                                , ( if
+                                        (periodicRun.status == JobStatus.JobRunning)
+                                    then
+                                        iconButton
+                                            "btn btn-default btn-xs"
+                                            "glyphicon glyphicon-stop"
+                                            "Stop and Delete Instance"
+                                    else
+                                        iconButton
+                                            "btn btn-default btn-xs"
+                                            "glyphicon glyphicon-trash"
+                                            "Delete Instance"
+                                    )
```

Comment:
@FRosner 
IMHO:
you trash the instance in any case, no matter what is its current status. So, trash should be shown to user, otherwise this "STOP" with side effect as really DELETE is misleading even more. 

Nonetheless, check this [fix](https://github.com/FRosner/cluster-broccoli/pull/354/commits/f1b634cfc8964e3dc0034f22745b0da3d7bfb66a)

Reply:
Thanks for the fix. Will a running periodic job that you stop also be completely removed or marked as dead and ready for garbage collection? If a periodic job which you invoke stop on is removed independently of its current status, we can indeed consider showing the trash can. But then we should show it always, even for running jobs.

Reply:
@FRosner ,
> Will a running periodic job that you stop also be completely removed or marked as dead and ready for garbage collection

With current version of Nomad it will be trashed. There is no option to just STOP it.

> But then we should show it always, even for running jobs

This was my initial point, but you should better align with other opinions.

In current state code satisfies requirements provided in the ticket.

Reply:
Ok. Given what you say I suggest we replace the stop button with a trash button completely, removing the condition on the job status. The title could say "Stop and delete periodic run" or something like this.

You can align with Nico but to me this sounds like the best option.

Sorry for going back-and-forth but I found the discussion super useful! 👍

Reply:
Having said that I am also fine with the current implementation and happy to merge it like this. You tell me how you guys decided.

Reply:
@FRosner 
Regarding 
> You can align with Nico but to me this sounds like the best option.
> Sorry for going back-and-forth but
 
Btw, Noco is not in the office. There are requirements in the ticket, which was actually refined as well. The code satisfies this requirements and from my point of view there is no reason not to take it for next steps.

In case this finding will be reviewed - it should end up as a new ticket. In any case it is not to much work.

Reply:
Works for me!